src/main-process/parse-command-line.js

```diff
@@ -134,6 +138,18 @@ module.exports = function parseCommandLine (processArgs) {
     devMode = true
   }
 
+  let projectSpecification = {}
+  if (projectSpecificationFile) {
+    const contents = Object.assign({}, readProjectSpecificationSync(projectSpecificationFile, executedFrom))
+    const config = contents.config
+    const originPath = projectSpecificationFile
```

Comment:
this one too, unless it's clearer to call this `originPath` in the next line as well